lib/tpm-support-lib.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Documentation       Collection of keywords related to UEFI TPM support
+
+Library             ./tpm-support-lib.py
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+
+*** Keywords ***
+# TODO: Most options are split in 2 lines either because name is too long or
+#    they have too many arguments. Get TCG2 Menu Construction currently
+#    returns those as different list items
+Get TCG2 Menu Construction
+    [Documentation]    Get TCG2 Menu Construction.
+    [Arguments]    ${checkpoint}=Esc=Exit
+    @{menu}=    Create List
+    @{additional_remove}=    Create List
+    ...    TPM2 Physical Presence Operation
+    ...    TCG2 Protocol Configuration
+    # Read screen and go to next page
+    FOR    ${i}    IN RANGE    3
+        ${out}=    Read From Terminal Until    ${checkpoint}
+        ${menu2}=    Parse TCG2 Menu Snapshot Into Construction
+        ...    ${out}
+        ...    ${additional_remove}
+        ${menu}=    Combine Lists    ${menu}    ${menu2}
+        Press Key N Times    1    ${PAGEDOWN}
+    END
+    # Go back to top
+    FOR    ${i}    IN RANGE    3
+        Sleep    0.1
+        Press Key N Times    1    ${PAGEUP}
+    END
+    ${menu_dict}=    Convert List Of Pairs Into Dictionary    ${menu}
+    RETURN    ${menu_dict}
+
+Enter TCG2 Menu
+    [Documentation]    This keyword enters TCG2 menu after the platform was powered on.
```

Comment:
@m-iwanicki please wrap at 80 signs

Reply:
fixed in [ca43cb6](https://github.com/Dasharo/open-source-firmware-validation/pull/164/commits/ca43cb6284650227c50aca4e3e4cc2c83598b05c)